fixtures/flows/login/actions.js

```diff
@@ -1,38 +1,47 @@
-export const buyerLogin = async (email, password) => {
-    if (email == undefined) {
-        email = process.env.BUYER_EMAIL;
-    }
-    if (password == undefined) {
-        password = process.env.BUYER_PASSWORD
-    }
-    await login(email, password);
-};
+import * as util from '../../flows/utils'
 
-export const sellerLogin = async (email, password) => {
-    if (email == undefined) {
-        email = process.env.SELLER_EMAIL;
-    }
-    if (password == undefined) {
-        password = process.env.SELLER_PASSWORD
-    }
-    await login(email, password);
-};
+/* eslint-disable no-param-reassign */
```

Comment:
is this still needed? if so can it be put closer to where the line is with `eslint-disable-next-line`